docs/source/features/metrics.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Monitoring and Metrics
+description: Understand Apollo Server's performance
+---
+
+Understanding the behavior of GraphQL execution inside of Apollo Server is critical to developing and running a production GraphQL layer. Apollo Server enables GraphQL monitoring in Apollo Engine and provides more primitive native mechanisms to log each phase of a GraphQL request.
+
+## Apollo Engine
+
+Apollo Engine provides an integrated hub for all GraphQL performance data that is free for one million queries per month. With an API key from the [Engine UI](https://engine.apollographql.com/), Apollo Server reports performance and error data out-of-band. Apollo Engine then aggregates and displays information for [queries](https://www.apollographql.com/docs/engine/query-tracking.html), [requests](https://www.apollographql.com/docs/engine/performance.html), the [schema](https://www.apollographql.com/docs/engine/schema-analytics.html), [errors](https://www.apollographql.com/docs/engine/error-tracking.html). In addition to aggregating data, Apollo Server provides [proactive alerts](https://www.apollographql.com/docs/engine/alerts.html), [daily slack reports](https://www.apollographql.com/docs/engine/reports.html), and [Datadog integration](https://www.apollographql.com/docs/engine/datadog.html).
+
+To set up Apollo Server with Engine, [click here](https://engine.apollographql.com/) to get an Engine API key and provide it to the `ENGINE_API_KEY` environment variable. Setting an environment variable can be done in commandline(`ENGINE_API_KEY=REPLACE_WITH_YOUR_API_KEY node start-server.js`) or with the [dotenv npm package](https://www.npmjs.com/package/dotenv).
```

Comment:
Is this how setting the key via the command line actually works?

`ENGINE_API_KEY=REPLACE_WITH_YOUR_API_KEY node start-server.js` . How does putting the engine key before node work?